server/database/queries/signIn.js

```diff
@@ -0,0 +1,11 @@
+const dbConnection = require('../connection');
+
+const signIn = (email) => {
+  const sql = {
+    text: 'SELECT * FROM users where email = $1 ;',
+    values: [email],
+  };
+  return dbConnection.query(sql);
+};
+
+module.exports = signIn;
```

Comment:
This query is not necessary , you can use getClientDataByEmail and add the password, or if you want to keep it this way, it is better to name the function checkEmail for example, instead of signIn